modules/release-notes/pages/0.7.0-rn.adoc

```diff
@@ -85,7 +85,7 @@ To run a `+dfx+` project using an emulator instead of the replica, you can now r
 +
 This decouples the version of `+dfx+` from the release version of `+agent-js+`. Additionally, it allows for debug builds to install properly (as `+dfx+` has a non-release version).
 * You can now install the DFINITY Canister SDK on for M1 Macs running Darwin. 
-* Allow upload of assests of any size from the asset canister
+* Allow upload of assets of any size from the asset canister
 + The following asset canister call methods are added:
```

Comment:
This `+` appears in the final render. I _think_ there should be a line break here.

Reply:
I think you are correct

Reply:
Added.